base/floatfuncs.jl

```diff
@@ -223,7 +223,7 @@ end
 
 # isapprox: approximate equality of numbers
 """
-    isapprox(x, y; rtol::Real=atol>0 ? 0 : √eps, atol::Real=0, nans::Bool=false, norm::Function)
+    isapprox(x, y; atol::Real=0, rtol::Real=atol>0 ? 0 : √eps, nans::Bool=false[, norm::Function])
```

Comment:
```suggestion
    isapprox(x, y; atol::Real=0, rtol::Real=atol>0 ? 0 : √eps, nans::Bool=false, norm::Function=abs)
```

Reply:
`=norm` would be more accurate, since this signature is for the general `x,y` case, not specifically for numbers.

Reply:
This concerns a weird edge case. `isapprox` is originally defined in `Base` and then extended in `LinearAlgebra` (at least as far as I can tell). The docstring is in `Base`, but also covers partly the signature of the method defined in `LinearAlgebra`. Therefore I think both `norm=abs` and `norm=norm` are misleading for the casual reader. Therefore, I decided against specifying the default value in the signature and instead just mark it as optional, which should tell the inclined reader that here is probably more to it in the textual description.

The only other sane option IMHO would be to add a *second* function signature to the docstring, i.e,
```julia
    isapprox(x::Number, y::Number; atol::Real=0, rtol::Real=atol>0 ? 0 : √eps, nans::Bool=false, norm::Function=abs)
    isapprox(x::AbstractArray, y::AbstractArray; atol::Real=0, rtol::Real=atol>0 ? 0 : √eps, nans::Bool=false, norm::Function=norm)
```

However, this has the downside that it looks *horribly complicated* for what I consider a relatively straightforward (and at the same time often-used) function. Therefore, I still favor my original proposal to mark `norm::Function` as optional and to give the default values only in the description.

Reply:
norm=norm is fine in the sense that `norm(::Number)` is defined as `abs`.

Reply:
Functionally it is the same, I agree. However, I'd assume that it is not reasonable to use functionality from `LinearAlgebra` (`norm(...)` is defined there) in `Base`, and I also do not see the benefit of having the documentation specifically explaining this mismatch of documentation and implementation. Thus I will mark this resolved for now, as I feel like there are no strong feelings either way.

However, since you are the much more experienced Julia developer, I will of course change it if you think it is necessary.